src/components/RadioButtonGroup/styles.scss

```diff
@@ -0,0 +1,146 @@
+/* stylelint-disable no-descending-specificity,max-line-length */
+@import '../../styles/colors';
+@import '../../styles/shadows';
+@import '../../styles/margins';
+@import '../../styles/paddings';
+@import '../../styles/font-sizes';
+@import '../../styles/border-radius';
+
+%border-margin {
+    border-radius: 0;
+    border-width: 1px;
+    margin-left: -1px;
+}
+
+@mixin user-select() {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+.rainbow-radio-button-group_container {
+    position: relative;
+}
+
+.rainbow-radio-button-group_label {
+    display: inline-block;
+    color: $color-gray-4;
+    font-size: $font-size-text-medium;
+    margin-right: $margin-small;
+    margin-bottom: $margin-xx-small;
+}
+
+.rainbow-radio-button-group_inner-container {
+    // display: inline-block;
+    border-radius: 1.25rem;
+    border: solid 1px #e3e5ed;
+    background-color: rgba(227, 229, 237, 0.4);
+    padding-top: calc(0.1rem + 1px);
```

Comment:
We should not align the content with `padding-bottom` an  `padding-top`, instead, like we already use `flex`  we should use `align-items: center` and `height: 2.5rem`